types/v56/constants.go

```diff
@@ -90,6 +90,10 @@ const (
 	MimeMetaData = "application/vnd.vmware.vcloud.metadata+xml"
 	// Mime for metadata value
 	MimeMetaDataValue = "application/vnd.vmware.vcloud.metadata.value+xml"
+	// Mime for a admin network
+	MimeExtensionNetwork = "application/vnd.vmware.admin.extension.network+xml"
+	// Mime for for a external network
```

Comment:
Typo/grammar, suggested fix:
`// Mime for an external network`

Reply:
fixed

Reply:
Almost, "a" -> "an"

Reply:
done